app/services/katello/pulp3/repository/yum.rb

```diff
@@ -149,6 +149,16 @@ def copy_units(source_repository, content_unit_hrefs, dependency_solving)
           tasks
         end
 
+        def copy_all(source_repository)
+          data = PulpRpmClient::Copy.new
+          data.config = [{
+            source_repo_version: source_repository.version_href,
+            dest_repo: repository_reference.repository_href
+          }]
+
+          [api.copy_api.copy_content(data)]
```

Comment:
Is this return value used anywhere?  why an array?

Reply:
copy_all 's return tries to mimic the return of copy_content_from_source in the same file for consistency.